package/system/fstools/patches/010-glob-onlydir.patch

```diff
@@ -0,0 +1,158 @@
+From patchwork Wed Mar 25 23:27:47 2020
+Content-Type: text/plain; charset="utf-8"
+MIME-Version: 1.0
+Content-Transfer-Encoding: 7bit
+X-Patchwork-Submitter: Rosen Penev <rosenp@gmail.com>
+X-Patchwork-Id: 1261704
+Return-Path: <openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org>
+X-Original-To: incoming@patchwork.ozlabs.org
+Delivered-To: patchwork-incoming@bilbo.ozlabs.org
+Authentication-Results: ozlabs.org; spf=pass (sender SPF authorized)
```

Comment:
There is 100+ lines of SMTP protocol chatter, quite unncessary.

Reply:
And there should naturalyl be also a version bump for fstools.

Reply:
Replaced with openembedded patch. And done.